docs/application/web/guides/machine-learning/trainer.md

```diff
@@ -0,0 +1,294 @@
+    layer_pooling2d2           pooling2d          1:32:26:26 layer_conv2d1/activ
+--------------------------------------------------------------------------------
+      layer_flatten3             flatten          1:32:25:25    layer_pooling2d2
+--------------------------------------------------------------------------------
+           layer_fc4     fully_connected         1:1:1:20000      layer_flatten3
+--------------------------------------------------------------------------------
+ layer_fc4/activatio          activation           1:1:1:100           layer_fc4
+--------------------------------------------------------------------------------
+           layer_fc5     fully_connected           1:1:1:100 layer_fc4/activatio
+--------------------------------------------------------------------------------
+      cross_softmax0       cross_softmax            1:1:1:10           layer_fc5
+================================================================================
+```
+
+### Create model from a configuration file
+
+Configuration file structure is described in [the guide for Tizen Native API](../../../native/guides/machine-learning/machine-learning-training#create-a-model-from-ini-formatted-file). This is a simplified configuration for the model created in the previous section:
+
+```ini
+[model]
+type = NeuralNetwork
+batch_size = 32
+epochs = 2
+
+[optimizer]
+type = sgd
+learning_rate = 0.01
+
+[layer_input0]
+type = input
+input_shape = 1:28:28
+
+[layer_conv2d1]
+type = conv2d
+filters = 32
+kernel_size = "3, 3"
+activation = relu
+
+[layer_pooling2d2]
+type = pooling2d
+pooling = max
+pool_size = "2, 2"
+
+[layer_flatten3]
+type = flatten
+
+[layer_fc4]
+type = fully_connected
+unit = 100
+activation = relu
+
+[layer_fc5]
+type = fully_connected
+unit = 10
+activation = softmax
+```
+
+To load the configuration use `createModel` function:
+
+```javascript
+var configFile = "documents/model.ini";
+var model = tizen.ml.trainer.createModel(configFile);
+```
+
+Compile the model:
+
+```javascript
+model.compile({
+    loss: "cross",
+});
+```
+
+## Training
+
+Connect the dataset to the model:
+
+```javascript
+model.setDataset(dataset);
+```
+
+Now the model is ready to train. To do that use `run` method:
+
+```javascript
+var options = {
+    epochs: 2,
+    batch_size: 32,
+};
+
+function successCallback() {
+    console.log("done");
+}
+
+function errorCallback(error) {
+    console.log(error);
+}
+
+model.run(options, successCallback, errorCallback);
+```
+
+## Save trained model to a file
+
+The trained model can be saved to a file for future use. Let's use the `FORMAT_INI_WITH_BIN` format, which will produce two files: an `ini` file with the network structure and a binary file with the weights. There are more formats which can be found in [API reference](../../api/7.0/device_api/mobile/tizen/ml_trainer.html#Model::saveToFile). Remember to add proper privileges to the `config.xml` as described in [the prerequisites](#prerequisites):
```

Comment:
Change has been suggested to maintain consistency with other API links on this page.
```suggestion
The trained model can be saved to a file for future use. Let's use the `FORMAT_INI_WITH_BIN` format, which will produce two files: an `ini` file with the network structure and a binary file with the weights. There are more formats which can be found in [API reference](../../api/latest/device_api/mobile/tizen/ml_trainer.html#Model::saveToFile). Remember to add proper privileges to the `config.xml` as described in [the prerequisites](#prerequisites):
```

Reply:
Sorry, I missed these links.